signac/common/host.py

```diff
@@ -38,6 +35,7 @@ def get_default_host(config=None):
             raise ConfigError("No hosts specified.")
 
 
+# deprecated
```

Comment:
I think I'd prefer to add one comment to the top of this module that the whole module is deprecated instead of commenting on each private function here. The per-function comments are better applied where we don't deprecate the whole module.